.github/workflows/ci.yml

```diff
@@ -29,25 +29,25 @@ jobs:
         ./ci/do_ci.sh cmake.test
 
   cmake_gcc_maintainer_sync_test:
-    name: CMake gcc 13 (maintainer mode, sync)
-    runs-on: ubuntu-latest
```

Comment:
When will ubuntu-latest be updated to point to 24.04?

Reply:
No, ubuntu-latest still points to ubuntu 22.04.

What changed and got broken is ubuntu 22.04

Ubuntu 24-04 is still beta, so latest will point to that later.